cfgov/cfgov/settings/local.py

```diff
@@ -82,9 +88,6 @@
         'SHOW_TOOLBAR_CALLBACK': lambda request: DEBUG,
     }
 
-
-MIDDLEWARE_CLASSES += CSP_MIDDLEWARE_CLASSES
```

Comment:
We should always include `CSP_MIDDLEWARE`

```suggestion
MIDDLEWARE += CSP_MIDDLEWARE
```

Reply:
We can save this for a later PR.